package.json

```diff
@@ -1,29 +1,28 @@
 {
   "name": "@fiverr/i18n",
-  "version": "1.5.1",
+  "version": "1.5.2",
   "description": "Translation helper",
   "author": "Fiverr dev team",
   "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/fiverr/i18n.js.git"
-  },
-  "bugs": {
-    "url": "https://github.com/fiverr/i18n.js/issues"
-  },
   "homepage": "https://github.com/fiverr/i18n.js#readme",
   "engines": {
     "node": ">=6.9.4"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fiverr/i18n.js.git"
+  },
   "main": "index.js",
   "scripts": {
     "prepublishOnly": "npm t && npm run lint",
     "test": "./node_modules/mocha/bin/mocha -- tests/*.js --inspect",
     "lint": "./node_modules/eslint/bin/eslint.js -c .eslintrc **/*.js --quiet"
   },
   "dependencies": {
-    "@fiverr/futile": "^1.10.0",
-    "deep-freeze": "0.0.1"
+    "deep-freeze": "0.0.1",
+    "lodash.get": "^4.4.2",
```

Comment:
Why not use our vendors-lodash module?

Reply:
This repo is public. Fiverr's vendors collections are private

Reply:
👍